doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,65 @@
+
+Let's consider classical input data consisting of :math:`M` examples, with :math:`N` features each, 
+
+.. math:: \mathcal{D}=\{x^{(1)}, \ldots, x^{(M)}\},
+
+where :math:`x^{(m)}` is a :math:`N`-dimensional vector for :math:`m=1,\ldots,M`. To embed this data into :math:`n` quantum subsystems (:math:`n` qubits *or* :math:`n` qumodes for discrete- and continuous-variable quantum computing, respectively), we can use various embedding techniques, some of which are explained briefly below. 
+
+
+Basis Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+Basis embedding associates each input with a computational basis state of a qubit system. Therefore, classical data has to be in the form of binary strings. The embedded quantum state is the bit-wise translation of a binary string to the corresponding states of the quantum subsystems. For example, :math:`x=1001` is represented by the 4-qubit quantum state :math:`\mid 1001 \rangle`. Hence, one bit of classical information is represented by one quantum subsystem.
+
+Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. For basis embedding, each example has to be a N-bit binary string; :math:`x^{(m)}=(b_1,\ldots,b_N)` with :math:`b_i \in \{0,1\}` for :math:`i=1,\ldots,N`. Assuming all features are repesented with unit binary precision (one bit), each input example :math:`x^{(m)}` can be directly mapped to the quantum state :math:`\mid x^{(m)}\rangle`. **This means that the number of quantum subsystems,** :math:`\bm{n}` **, must be at least equal to** :math:`\bm{N}`. An entire dataset can be represented in superpositions of computational basis states as
+
+
+.. math:: \mid \mathcal{D} \rangle = \frac{1}{\sqrt{M}} \sum_{m=1}^{M} |x^{(m)} \rangle.
+
+For example, let's say we have a classical dataset containing two examples :math:`x^{(1)}=01` and :math:`x^{(2)}=11`. The corresponding basis encoding uses two qubits to represent :math:`\mid x^{(1)} \rangle=|01 \rangle` and :math:`\mid x^{(2)} \rangle=|11 \rangle` resulting in
+
+.. math:: \mid \mathcal{D} \rangle = \frac{1}{\sqrt{2}}|01 \rangle + \frac{1}{\sqrt{2}} |11 \rangle.
+
+.. note:: For :math:`N` bits, there are :math:`2^N` possible basis states. Given :math:`M \ll 2^N`, the basis embedding of :math:`\mathcal{D}` will be sparse :cite:`schuld2018supervised`. 
+
+
+Amplitude Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+In the amplitude embedding technique, data is encoded into the amplitudes of a quantum state. A classical **normalized** :math:`N`-dimensional datapoint :math:`x` is represented by the amplitudes of a :math:`n`-qubit quantum state :math:`\mid \psi_x \rangle` as
```

Comment:
```suggestion
In the amplitude-embedding technique, data is encoded into the amplitudes of a quantum state. A classical **normalized** :math:`N`-dimensional datapoint :math:`x` is represented by the amplitudes of a :math:`n`-qubit quantum state :math:`\mid \psi_x \rangle` as
```

Reply:
I am curious as to why we are changing the format now? Should I change this for basis embedding too? also in the headings ?

Reply:
It's not a format change, it's just correcting the grammar. When a compound adjective appears before the known, you need to hyphenate it. If it appears on it's own (e.g., the section title) or after the noun, you don't hyphenate
https://www.grammar.com/compound-adjectives/

Reply:
oh wow. I did not even think about that! really cool! :)